Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -3061,10 +3051,10 @@ instructionLabel(_struct_new)
     loadp IPInt::StructNewMetadata::typeIndex[MC], a1  # type index
     move sp, a2
     operationCallMayThrow(macro() cCall3(_ipint_extern_struct_new) end)
-    loadh IPInt::StructNewMetadata::params[MC], t0  # number of parameters popped
-    mulq StackValueSize, t0
-    addq t0, sp
-    pushQuad(r1)
+    loadh IPInt::StructNewMetadata::params[MC], t1  # number of parameters popped
```

Comment:
We should use `r1` here to ensure x86_64 doesn't break.

Reply:
Are you sure about this? I couldn't quite follow this since t1 is used as a temp here.

Reply:
Agh sorry, yes, this usage is fine. Been reading too much ARM and got dst order mixed up again.